skyportal/handlers/api/photometric_series.py

```diff
@@ -0,0 +1,1764 @@
+                This is useful if you have an HDF5 file downloaded
+                from the SkyPortal backend, and want to associate it
+                with a PhotometrySeries object.
+                We use an MD5 hash of the file contents.
+            - in: query
+              name: sortBy
+              nullable: true
+              default: obj_id
+              schema:
+                type: string
+              description: |
+                The field to sort by. Currently allowed options are ["id", "ra", "dec", "redshift", "saved_at"]
+            - in: query
+              name: sortOrder
+              nullable: true
+              default: asc
+              schema:
+                type: string
+              description: |
+                The sort order - either "asc" or "desc". Defaults to "asc"
+            - in: query
+              name: numPerPage
+              nullable: true
+              schema:
+                type: integer
+              description: |
+                Number of sources to return per paginated request.
+                Defaults to 100. Max 500.
+            - in: query
+              name: pageNumber
+              nullable: true
+              schema:
+                type: integer
+              description: Page number for paginated query results. Defaults to 1
+          responses:
+            200:
+              content:
+                application/json:
+                  schema:
+                    allOf:
+                      - $ref: '#/components/schemas/Success'
+                      - type: object
+                        properties:
+                          data:
+                            type: object
+                            properties:
+                              series:
+                                type: array
+                                items:
+                                  $ref: '#/components/schemas/PhotometricSeries'
+                              totalMatches:
+                                type: integer
+                              pageNumber:
+                                type: integer
+                              numPerPage:
+                                type: integer
+        """
+        if photometric_series_id is not None:
+            with self.Session() as session:
+                ps = session.scalars(
+                    PhotometricSeries.select(self.current_user).where(
+                        PhotometricSeries.id == photometric_series_id
+                    )
+                ).first()
+                if ps is None:
+                    return self.error('Invalid photometric series ID.')
+                return self.success(data=ps.to_dict())
+
+        # get all photometric series
+        ra = self.get_query_argument('ra', None)
+        dec = self.get_query_argument('dec', None)
+        radius = self.get_query_argument('radius', None)
+        object_id = self.get_query_argument('objectID', None)
```

Comment:
how is `objectID` different than `Obj` or `Source`?

Reply:
I think in this case it is the id of the skyportal Obj (or Source)